faq.md

```diff
@@ -63,6 +63,16 @@ Précision concernant les bots discords : toute demande d'aide concernant de pr
 
 Précision concernant les bases de données : la section #database concerne principalement les questions relatives aux schémas et aux requêtes. Les questions concernant les bibliothèques spécifiques à leur langages de programmation n'y ont pas leur place.
 
+**Sécurité informatique**
+
+Parce que sur ce serveur il y a aussi beaucoup de novices en informatique, nous prenons avec des pincettes tout ce qui touche à la sécurité informatique. Ce sujet n'est abordé *que* lorsqu'il s'agit de renforcer la protection d'un système ou bien de corriger des failles évidentes (exemple: injection sql).
+
+Il n'y a personne qui est à la fois habitué, experte reconnue du domaine et qui voudrait devenir ambassadeur d'un canal sécurité. De ce fait nous ne pouvons pas garantir que les discussions concernant le *pentesting*, le *reverse engineering* et d'autres formes de crack/hack se déroule dans un cadre déontologique. Ces discussions ont interdites sur NaN.
+
+**Web-scrapping**
+
+TODO: ce qui serait cool (mais très long) c'est de parcourir les CGU pour retrouver pour chacun de ces sites: Amazon, YouTube, Instagram, Facebook, Twitter où on dit "interdit de selenium-iser notre site, passez par l'api"
+
```

Comment:
@florentm35 gros manque d'inspiration :sweat:

Reply:
ah et donc pour chacun de ces sites on fait un lien markdown genre `[youtube](https://youtube.com/la-cgu#la-regle-qui-dit-tu-peux-pas)`

Reply:
J'aurais plutot mis un truc:
"Le scrapping de donnees étant cadre légale douteux, il s'agit d'un sujet prohibé sur NaN [plus d'infos]<https://lexing.be/le-scraping-est-il-legal/>"

Reply:
https://github.com/NotANameServer/discord/commit/c8ce2f1d004e3d8bb8865cd4a5e7b50c93fc8de1